qualtran/cirq_interop/_cirq_interop.py

```diff
@@ -155,6 +164,13 @@ def _split_qvars_for_regs(
     return qvars_regs
 
 
+@attr.frozen
+class QReg:
```

Comment:
consider making this private (?)

If private: add a short docstring about how it's used.

If public: add a long docstring about how it's meant to be used

Reply:
SG, will make it private.